js/ui/popover.js

```diff
@@ -69,7 +70,12 @@ const getEventDelay = function(that, optionName) {
 const attachEvent = function(that, name) {
     const target = that.option('target');
     const isSelector = isString(target);
-    const event = getEventName(that, name + 'Event');
+    let event = getEventName(that, name + 'Event');
+
+    if(that.option('shading') && event === 'mouseleave') {
```

Comment:
Is problem actual only for `mouseleave` hideEvent? What's about `mouseout`? To my mind we should ignore all custom hideEvents when shading is enabled

Reply:
I agree with @ksercs . Look wider :)

Reply:
Yeap, now all events will be ignored , except of 'click' and 'dxclick'